test/selenium/util/PayPalCheckout.scala

```diff
@@ -0,0 +1,49 @@
+package selenium.util
```

Comment:
I think this belongs in pages, not util - what do you think @Ap0c @svillafe?

Reply:
I put it in util, because it is an object that is being used in other two pages.